Description: Since gogo/protobuf 1.2 unknown fields in proto3 are preserved.
 This results in a XXX_unrecognized of type []byte to be created in the struct.
 As a side effect the struct can't be used as a map key anymore, resulting in
 a libnetwork build failure.

 Fix this by setting goproto_unrecognized_all=false which suppresses this field.

 Upstream doesn't have the problem because they have pinned an older version
 of gogo/protobuf: https://github.com/docker/libnetwork/pull/2242
Author: Felix Geyer <fgeyer@debian.org>

Index: docker/libnetwork/agent.proto
===================================================================
--- docker.orig/libnetwork/agent.proto
+++ docker/libnetwork/agent.proto
@@ -10,6 +10,7 @@ option (gogoproto.stringer_all) = true;
 option (gogoproto.gostring_all) = true;
 option (gogoproto.sizer_all) = true;
 option (gogoproto.goproto_stringer_all) = false;
+option (gogoproto.goproto_unrecognized_all) = false;
 
 // EndpointRecord specifies all the endpoint specific information that
 // needs to gossiped to nodes participating in the network.
